### PR TITLE
Add docs about building the docs.

### PR DESCRIPTION
**Pull request by douglatornell**
re: #10895

### CODE REVIEW

**Comment by jzf2101** [comment]
@douglatornell can you explain why we now need sphinx_rtd_theme in addition to build?

**Comment by douglatornell** [comment]
@jzf2101 sphinx_rtd_theme is a new dependency because it used to be bundled with sphinx, but a while ago they stopped doing that, so now it has to be installed separately.

I'll add narrative about the sphinx build on travis-ci (pr #10911)

**Comment by jzf2101** [comment]
could you reverence cleaning builds

**Comment by jzf2101** [comment]
Thanks!

**Comment by douglatornell** [comment]
@jzf2101 Added notes about docs in Travis-CI, and how to do clean builds.

**Review by jzf2101 — APPROVED** [review]

**Comment by Carreau** [comment]
Great awesome ! Merging and I'll add an extra comment inline to get your input on a potential extra PR !

File: docs/README.rst [hunk 1]
```diff
@@ -12,13 +13,29 @@ Pull requests.
...
-    sudo apt-get install python-sphinx
+    sudo apt-get install python3-sphinx python3-sphinx-rtd-theme
+
+In a conda environment, you can use::
+
+    conda install sphinx sphinx_rtd_theme
+
+In a Python 3 ``venv``, you should be able to run::
+
+    pip install -U sphinx sphinx_rtd_theme
```
Comment (Carreau) [inline docs/README.rst]: I think there is a `docs/requirements.txt`, maybe `pip install --update -r docs/requirements.txt` is even more general ? But I think that's already an improvement as is so I already merged ! Thanks !

Comment (douglatornell) [reply docs/README.rst]: Good point! I should have thought of that in light of 326f41b and a couple of other things that happened during the sprint yesterday. I'll work on another PR for this.

Comment (jzf2101) [reply docs/README.rst]: 👍

**Pull request merged**